### PR TITLE
Adb/config/ts

### PR DESCRIPTION
**Pull request by adbarc92**

### CODE REVIEW

File: src/server/index.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+app.get(
+  '/qa/questions/:question_id/answers',
+  (req: Request, res: Response) => {},
+);
+/* Add a Question */
+app.post('/qa/questions', (req: Request, res: Response) => {});
+/* Add an Answer */
+app.post(
+  '/qa/questions/:question_id/answers',
+  (req: Request, res: Response) => {},
```
Comment (alishaedington) [inline src/server/index.ts]: why is it written out in this way? req: Request, res: Response

File: src/server/db/index.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+// const { Pool, Client } = require('pg');
+import { Pool, Client } from 'pg';
```
Comment (alishaedington) [inline src/server/db/index.js]: are you using Client anywhere? I'm not seeing it..

**Review by alishaedington — APPROVED** [review]
I assume everything is there for a reason, just a few small questions

File: src/server/index.ts (hunk 1 above)
Comment (adbarc92) [reply src/server/index.ts]: This is how type-checking is done with TypeScript. `Request` and `Response` are the types of each respective variable.

File: src/server/db/index.js (hunk 2 above)
Comment (adbarc92) [reply src/server/db/index.js]: Good point. Removing

**Pull request merged**